src/main/resources/init.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TYPE public.ticket_type AS ENUM (
+    'DAY',
+    'WEEK',
+    'MONTH',
+    'YEAR'
+);
+
+CREATE TABLE public."user_data" (
```

Comment:
do not use quotes in table names and `public`. just
```sql
CREATE TABLE user_data (...)
```

Reply:
Removed quotes in table names and public.